.travis.yml

```diff
@@ -11,17 +11,18 @@ env:
     - COMPOSER_OPTIONS="--prefer-stable"
 
 php:
-  - 5.4
```

Comment:
Please keep the existing build jobs for now. I will consider which PHP versions I want to support during the next days. But as long as the Composer constraints do not limit the usable PHP versions I do not want to not test them anymore.